mvn install fails

**Describe the bug**
 When I tried doing mvn install, I get the following test case failure. 
 
 ```[ERROR] Failures:
[ERROR]   ClientBuilderTest.testInvalidKubeconfig:145
Expected: is "http://localhost:8080"
     but: was "https://0F9B61E06182A43814EB425AFD927EFC.gr7.us-east-2.eks.amazonaws.com"
[INFO]
[ERROR] Tests run: 207, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Kubernetes Client API 12.0.1-SNAPSHOT:
[INFO]
[INFO] Kubernetes Client API .............................. SUCCESS [  0.366 s]
[INFO] client-java-api .................................... SUCCESS [  9.143 s]
[INFO] client-java-proto .................................. SUCCESS [  4.498 s]
[INFO] client-java ........................................ FAILURE [02:48 min]```

** Client Version **
e.g. `11.0.1`

** Kubernetes Version **
e.g. `1.19.3`

** Java Version **
e.g. Java 8

**To Reproduce**
git clone --recursive https://github.com/kubernetes-client/java
cd java
mvn installr:

**Expected behavior**
It should finish successfully

**KubeConfig**
If applicable, add a KubeConfig file with secrets redacted.

**Server (please complete the following information):**
 - OS: [e.g. Linux]
 - Environment [e.g. container]
 - Cloud [e.g. Azure]

**Additional context**
Add any other context about the problem here.

--- a/util/src/test/java/io/kubernetes/client/util/ClientBuilderTest.java
+++ b/util/src/test/java/io/kubernetes/client/util/ClientBuilderTest.java
@@ -137,6 +137,7 @@ public class ClientBuilderTest {
   public void testInvalidKubeconfig() throws Exception {
     String path =
         withEnvironmentVariable("KUBECONFIG", "/non-existent")
+            .and("HOME", "/none-existent")
             .execute(
                 () -> {
                   final ApiClient client = ClientBuilder.standard().build();

